README.md

```diff
@@ -10,3 +10,40 @@ Bom-Squad is a tool to help users source components for DIY SMT Eurorack modules
 -	track a personal inventory of components
 -	and version a personal inventory as changes are made, ensuring that mistakes are rare and easily fixed
  
+## Getting started
+
+### Installing dependencies
+
+The site is built using [Django](https://www.djangoproject.com/). In order to get your development environment set up, first ensure you have Python installed (version 3.8 or higher **\<TBC\>**), and run the following commands in the directory where you have cloned this repository:
```

Comment:
I had Python 3.6.9 installed and couldn't do the DB migration due to an error with the bundled SQLite version, so took a guess at what the minimum Python version is. If you know what the actual minimum Python version is @PleatherStarfish, please let me know and I'll update this!